Source/ConstraintViewDSL.swift

```diff
@@ -99,3 +99,84 @@ public struct ConstraintViewDSL: ConstraintAttributesDSL {
     }
     
 }
+
+
+#if os(iOS) || os(tvOS)
+@available(iOS 11.0, *)
+public extension ConstraintViewDSL {
+    
+    func toCenter() { view.snp.makeConstraints { make in make.center.equalToSuperview() } }
+       
+       func toTopCenter(_ inset: CGFloat = 0, toSafeArea: Bool = false) {
+           view.snp.makeConstraints { make in
+               guard let superview = view.superview else { return }
+
+               make.top.centerX.equalTo(toSafeArea ? superview.safeAreaLayoutGuide : superview).inset(inset)
+           }
+       }
+
+       func toBottomCenter(_ inset: CGFloat = 0, toSafeArea: Bool = false) {
+           view.snp.makeConstraints { make in
+               guard let superview = view.superview else { return }
+
+               make.bottom.centerX.equalTo(toSafeArea ? superview.safeAreaLayoutGuide : superview).inset(inset)
+           }
+       }
+
+       func toLeftCenter(_ inset: CGFloat = 0, toSafeArea: Bool = false) {
```

Comment:
I don't know about maintainers of this library, but i guess these functions should be applying constraints to leading or trailing, not left and right